### PR TITLE
Fix part of #1287: Fix lint issues in app module in the kotlin files (completedstorylist)

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes part of #1287
`completedstorylist` folder `ktlint` check

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt [hunk 1]
```diff
@@ -35,8 +37,16 @@ class CompletedStoryListFragment : InjectableFragment() {
...
   ): View? {
-    val args = checkNotNull(arguments) { "Expected arguments to be passed to CompletedStoryListFragment" }
-    val internalProfileId = args.getInt(COMPLETED_STORY_LIST_FRAGMENT_PROFILE_ID_KEY, -1)
-    return completedStoryListFragmentPresenter.handleCreateView(inflater, container, internalProfileId)
+    val args =
+      checkNotNull(arguments) { "Expected arguments to be passed to CompletedStoryListFragment" }
+    val internalProfileId = args
+      .getInt(
+        COMPLETED_STORY_LIST_FRAGMENT_PROFILE_ID_KEY, -1
+      )
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt]: Nit changes for better code readability:
```
    val args = checkNotNull(arguments) {
      "Expected arguments to be passed to CompletedStoryListFragment"
    }
    val internalProfileId = args.getInt(
      COMPLETED_STORY_LIST_FRAGMENT_PROFILE_ID_KEY, -1
    )
```

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, one nit change suggested.

File: app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragment.kt]: ok, Thanks for review.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks

**Review by veena14cs — APPROVED** [review]
LGTM.

**Comment by anandwana001** [comment]
> LGTM.

Thanks for the review :)

**Pull request merged**